addons/website/static/tests/tours/html_editor.js

```diff
@@ -207,17 +210,18 @@ tour.register('test_html_editor_scss', {
             },
         },
         {
-            content: "save the html editor",
+            content: "save the scss editor",
             extra_trigger: `div.ace_line:contains("${demoCssModif}")`,
             trigger: ".o_ace_view_editor button[data-action=save]",
         },
         {
             content: "wait for reload",
             trigger: ":not(.o_ace_view_editor)",
             run: function () {}, // it's a check
+            timeout: 30000,
         },
         {
-            content: "reset view (after reload, html editor should have been reopened where it was)",
+            content: "reset view (after reload, scss editor should have been reopened where it was)",
```

Comment:
The beast real name is "HTML/SCSS Editor" :D 
HTML Editor is its nickname.
Scss editor is not really a thing AFAIK, I wouldn't have change those lines :) kinda useless diff